src/cli/install.ps1

```diff
@@ -180,4 +176,27 @@ function Install-Bun {
   if(!$hasExistingOther) {
     Write-Output "To get started, restart your terminal/editor, then type `"bun`"`n"
   }
-}
\ No newline at end of file
+
+  $RegistryKey = "HKCU:\Software\Microsoft\Windows\CurrentVersion\Uninstall\Bun"  
+  New-Item -Path $RegistryKey -Force | Out-Null
+  New-ItemProperty -Path $RegistryKey -Name "DisplayName" -Value "Bun" -PropertyType String -Force | Out-Null
+  # New-ItemProperty -Path $RegistryKey -Name "DisplayVersion" -Value $DisplayVersion -PropertyType String -Force | Out-Null
```

Comment:
I commented this out because it doesn't get updated in `bun upgrade` and I didn't want it to be incorrect